tests/js-modules/modules.py

```diff
@@ -262,6 +262,10 @@ def test_npm_app(network, args):
         assert len(r.body.data()) == key_size // 8
         assert r.body.data() != b"\x00" * (key_size // 8)
 
+        r = c.post("/app/generateRsaKey", {"size": 2048})
+        assert r.status_code == http.HTTPStatus.OK, r.status_code
+        assert not all(v == b"\x00" for v in r.body.data())
```

Comment:
This test should probably try to actually parse the returned key, likely by adding some new functions to infra/crypto.py.